DopplerJobsServer/appsettings.json

```diff
@@ -83,26 +83,26 @@
   "RelayEmailSenderSettings": {
     "SendTemplateUrlTemplate": "https://api.dopplerrelay.com/accounts/{accountId}/templates/{templateId}/message",
     "ApiKey": "REPLACE_FOR_EMAIL_SENDER_API_KEY",
-    "AccountId": 73,
-    "AccountName": "fromdoppler",
-    "Username": "easla@fromdoppler.com",
+    "AccountId": 8325,
+    "AccountName": "doppler_relay",
+    "Username": "leve2support@fromdoppler.com",
```

Comment:
Remember to update the secrets in the Swarm repository. How will you do it without having downtime? 

We can talk about it. Let me know.